### PR TITLE
Item Requirements

### PR DESCRIPTION
**Pull request by piperlincoln**
This PR covers the following requirements in Pivotal and completes the five coding requirements for this iteration:

* 1.18
* 1.20
* 1.24
* 1.25

When merged, keep in mind this will not be fully formatted, so placement of boxes and such may look awkward until we refine the styling.

### CODE REVIEW

File: src/components/CollectionBox.css [hunk 1]
```diff
@@ -18,4 +18,3 @@
     padding: 10px;
     width: 100%;
   }
-
```
Comment (Taracuda) [inline src/components/CollectionBox.css:21]: Probably being picky, but can this extra line be removed?

File: src/components/PageLayoutComponent.tsx [hunk 2]
```diff
@@ -29,6 +30,9 @@ const PageLayoutComponent: React.FC = () => {
       collectionId: 0,
       collectionName: collectionTitle,
       isPrivate: checkedValue,
+      collectionPhoto: collectionPhoto,
+      items: []
+      // {itemDescription: "test", itemId: 230, itemTitle: "My Sweet Item"}
```
Comment (Taracuda) [inline src/components/PageLayoutComponent.tsx]: I don't think we need this commented out line anymore

File: src/components/CollectionBox.css (hunk 1 above)
Comment (piperlincoln) [reply src/components/CollectionBox.css:21]: To clarify, are you asking if something will break if it is removed?

Comment (Taracuda) [reply src/components/CollectionBox.css:21]: I just meant we can probably remove it. I doubt it would break anything since the pre-checks passed.

Comment (piperlincoln) [reply src/components/CollectionBox.css:21]: Oh, the red means that the extra line was removed in my commit! So it was there before and now it isn't.

**Review by Taracuda — APPROVED** [review]

**Comment by hsantos940** [comment]
Great work on covering the requirements. Everything looks good to me! I like the new login button.

File: src/Database/ItemStore.ts [hunk 3]
```diff
@@ -0,0 +1,25 @@
+import { createContext } from "react";
```
Comment (firicano33) [inline src/Database/ItemStore.ts:1]: I see that you have added the functions to create, update and delete items.

**Review by firicano33 — APPROVED** [review]
This looks great.  I wish I could code!!!

**Comment by conhur** [comment]
Looks good. I see that we can add up to 10 items.

**Pull request merged**